src/test/java/com/yourcompany/Pages/GuineaPigPage.java

```diff
@@ -49,7 +49,7 @@ public void submitComment(String text) {
 
         // Race condition for time to populate yourCommentsSpan
         WebDriverWait wait = new WebDriverWait(driver, 15);
-        wait.until(ExpectedConditions.textToBePresentInElement(yourCommentsSpan, text));
+        //wait.until(ExpectedConditions.textToBePresentInElement(yourCommentsSpan, text));
```

Comment:
if we comment this out should we just remove this line?

Reply:
Hey @joshmgrant, if you can delete this line and rebase, I think we should merge this